assets/css/styles.css

```diff
@@ -37,7 +37,7 @@ h1 {
 }
 
 h2 {
-  font-size: 30px;
```

Comment:
the h2 for the whole site is 30px, just for the feature section is overrides to 18px